src/WaterPoke.java

```diff
@@ -0,0 +1,70 @@
+package src;
+
+public class WaterPoke extends Pokemon {
```

Comment:
Ik zou zo min mogelijk afkorten in Classnames. Gewoon waterPokemon voluit schrijven.